Sources/Filters/Sources/PlaneSource/index.js

```diff
@@ -115,12 +261,20 @@ const DEFAULT_VALUES = {
 export function extend(publicAPI, model, initialValues = {}) {
   Object.assign(model, DEFAULT_VALUES, initialValues);
 
+  model.normal = [0, 0, 1];
+  model.center = [0, 0, 0];
+
   // Build VTK API
   macro.obj(publicAPI, model);
   macro.setGet(publicAPI, model, ['xResolution', 'yResolution']);
-  macro.setGetArray(publicAPI, model, ['origin', 'point1', 'point2'], 3);
+  macro.setGetArray(publicAPI, model, ['origin'], 3);
+  macro.getArray(publicAPI, model, ['point1', 'point2', 'normal', 'center'], 3);
+
   macro.algo(publicAPI, model, 0, 1);
   vtkPlaneSource(publicAPI, model);
+
+  publicAPI.setPoint1(model.point1);
+  publicAPI.setPoint1(model.point2);
```

Comment:
The second point is not being set here (just a copy-paste typo), should be publicAPI.setPoint**2**(model.point2);